### PR TITLE
refactor  the cvs reader/writer server 

### PR DESCRIPTION
**Pull request by nongfushanquan**

### CODE REVIEW

File: cmd/demoserver/kvdatamake.go [hunk 1]
```diff
@@ -119,15 +142,26 @@ func generateData(folderName string, recorders int) int {
 	return fileNum
 }
 
+func DataService(ctx context.Context) {
+	grpcServer := grpc.NewServer()
+	pb.RegisterDataRWServiceServer(grpcServer, NewDataRWServer(ctx))
+	lis, err := net.Listen("tcp", PORT)
+	if err != nil {
+		fmt.Println("error happened ")
```
Comment (amyangfei) [inline cmd/demoserver/kvdatamake.go:150]: Can use `log.Panic` to terminate process when error happens.

File: cmd/demoserver/kvdatamake.go [hunk 2]
```diff
@@ -119,15 +142,26 @@ func generateData(folderName string, recorders int) int {
...
+	}
+	err = grpcServer.Serve(lis)
+	if err != nil {
+		fmt.Printf("error is %f", err)
```
Comment (amyangfei) [inline cmd/demoserver/kvdatamake.go:154]: ditto

File: cmd/demoserver/kvdatamake.go [hunk 3]
```diff
@@ -14,6 +14,7 @@ import (
 	"time"
 
 	"github.com/hanfei1991/microcosm/pb"
+	"github.com/pingcap/log"
```
Comment (hanfei1991) [inline cmd/demoserver/kvdatamake.go:17]: please cite "github.com/pingcap/tiflow/dm/pkg/log"

File: cmd/demoserver/kvdatamake.go (hunk 1 above)
Comment (nongfushanquan) [reply cmd/demoserver/kvdatamake.go]: fix it ,replace it with the log system

File: cmd/demoserver/kvdatamake.go [hunk 4]
```diff
@@ -14,35 +14,60 @@ import (
 	"time"
 
 	"github.com/hanfei1991/microcosm/pb"
+	"github.com/pingcap/tiflow/dm/pkg/log"
+
```
Comment (liuzix) [inline cmd/demoserver/kvdatamake.go:18]:
```suggestion
```

**Review by liuzix — APPROVED** [review]

**Pull request merged**